java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -2869,9 +2870,14 @@ private static native long stringReplaceWithBackrefs(long columnView, String pat
 
   private static native long quantile(long cudfColumnHandle, int quantileMethod, double[] quantiles) throws CudfException;
 
-  private static native long rollingWindow(long viewHandle, int min_periods, long aggPtr,
-                                           int preceding, int following,
-                                           long preceding_col, long following_col);
+  private static native long rollingWindow(long viewHandle,
```

Comment:
Nit: would be nice to move `viewHandle` argument to its own line to be consistent with the arg-per-line of the other args.  This also makes it consistent with the declaration of `Table.rollingWindowAggregate`.